src/test/resources/cartridge/migrations/001_spase_test.lua

```diff
@@ -0,0 +1,32 @@
+return{
+
+    up = function()
+        local utils = require('migrator.utils')
+
+        local routerSessionContexts = box.schema.space.create('spaceTest', { if_not_exists = true })
```

Comment:
Don't this complex and meaningless name 'routerSessionContexts' ? use 'test' or something like that